db/migrations/011.do.add-reverted-column.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE active_deals
+ADD COLUMN reverted BOOLEAN NOT NULL DEFAULT FALSE;
```

Comment:
This can lead to unexpected behavior in the future, because the schema doesn't prevent us from inserting new active deals without a reverted property.

It would be safer to add this constraint:

```suggestion
ALTER TABLE active_deals
ADD COLUMN reverted BOOLEAN;

UPDATE active_deals SET reverted = FALSE;

ALTER TABLE active_deals
ALTER COLUMN reverted SET NOT NULL;
```

Reply:
Thanks for the suggestion :)